dashboard/lib/pd/survey_pipeline/daily_survey_parser.rb

```diff
@@ -0,0 +1,123 @@
+require_relative 'transformer.rb'
+
+module Pd::SurveyPipeline
+  class DailySurveyParser < TransformerBase
+    include Pd::JotForm::Constants
+
+    # Map question type to answer type
+    ANSWER_TYPE_MAPPING = {
+      TYPE_NUMBER => ANSWER_TEXT,
+      TYPE_TEXTBOX => ANSWER_TEXT,
+      TYPE_TEXTAREA => ANSWER_TEXT,
+      TYPE_RADIO => ANSWER_SINGLE_SELECT,
+      TYPE_DROPDOWN => ANSWER_SINGLE_SELECT,
+      TYPE_CHECKBOX => ANSWER_MULTI_SELECT,
+      TYPE_MATRIX => ANSWER_MULTI_SELECT,
+      TYPE_SCALE => ANSWER_SCALE
+    }.freeze
```

Comment:
Tests covering each of these mappings would be good.

Reply:
Moving these mappings to the centralized `jot_form/constants.rb`. Any change to this mapping should be intentional.